extensions/jira/src/components/FormParentDropdown.tsx

```diff
@@ -23,7 +23,11 @@ const FormParentDropdown = forwardRef(
           query,
         });
 
-        if (result && result.sections.length > 0) {
+        if (result && result.sections.length > 1) {
+          // Retrieve from Current Search
+          return result.sections[1].issues;
```

Comment:
(non-blocking) I am wondering about the question here: https://github.com/raycast/extensions/issues/12646#issuecomment-2275694286 and why can't we include both current and historic searches in the typeahead dropdown? Jira's TTL for historic issues should be anyways out-of-scope for Raycast. If including historic might make the list too convoluted (or too many items) then feel free to disregard this suggestion.

```suggestion
          // Retrieve unique issues from Current and History Search
          return [...new Map([...result.sections[1].issues, ...result.sections[1].issues].map(issue => [issue.id, issue])).values()];
```

Reply:
Thanks for the review!

History search is actually a subset of the Current Search

Current Search lists all the valid epics, while History Search lists only the valid epics that the user interacted with before.

Merging both Current and History and selecting the unique out of them should yeild exactly the same results as just the Current Search.

Let me know if that makes sense

Reply:
Ah ok, thanks for the clarification. Then it just makes sense to use Current. 

But are we assuming jira api won't always populate current issues? In that case it will aid future debugging to somehow differentiate issues (with a different icon color/section) when they are being populated from history. That way it would be easier to tell if this issue arises again rather than asking the user to run in development mode or trigger the api.

Or if jira api contractually should always have current items, then we can keep this dropdown contract simple by just populating current issues or an empty list otherwise (ignore history)

Reply:
My 2 cents: We could de-duplicate and combine both the lists. It allows people to see the epics they interact with at the top, and the remaining search below.

Reply:
> Or if jira api contractually should always have current items, then we can keep this dropdown contract simple by just populating current issues or an empty list otherwise (ignore history).

Yep this is the case, Jira APIs should always have current items (if the search query should indeed match some items). 

My initial implementation was to just keep current search only and ignore history altogether indeed, but I noticed one case where the extension makes use of the history results; this is at the very beginning when there's no search query to begin with.

This is a screenrecording that shows this behaviour better (also updated in the PR description)


https://github.com/user-attachments/assets/c6e0587b-b8f6-4b64-83cf-596427574b06

You'll notice that when we first open the dropdown, before we write or search for anything, the dropdown shows the list of history items (which is KAN-2 in this case). Once I start typing, the list is populated with current items (which is both KAN-1 and KAN-2).

If I remove history search altogether, I'll be changing the existing behaviour such that when the user opens the drop down, it'll show an empty list, and then populate the list with the user searches for something.

I'd rather keep my changes to the minimum, and only change the behaviour necessary to fix the bug. This is why I think it's better to:
- Use history search when the user initially opens the drop down (same as before)
- Use current search when the user searches for something (to fix the issue of missing epics).

Reply:
> My 2 cents: We could de-duplicate and combine both the lists. It allows people to see the epics they interact with at the top, and the remaining search below.

It's an interesting proposal, but it's leaning to be a new feature or an improvement. We might think of adding a UI change as well to separate the epics in history vs others.

I want though to keep this PR strictly to just fixing the bug of missing epics. Hence, if there are no major issues with this fix, I think I'll just ship it and followups can be addressed in future PRs.

Reply:
Ok so conventionally typeahead APIs should list first 'n' entries if no query is specified. And based on the screenshot it seems JIRA api made it complicated and listing the entries in history instead. I was under the impression it would list 'n' random entries in current. Thanks for the explanation and I agree we should not break existing behavior in that case and keep history intact. You can resolve this issue.

Reply:
👍  to shipping it as it is.